src/test/kotlin/org/prebid/cache/functional/StorageSpec.kt

```diff
@@ -278,4 +279,46 @@ class ModuleStorageSpec : ShouldSpec({
             exception.responseBody should beEmpty()
         }
     }
+
+    should("throw an exception when ttlsecond is zero") {
+        //given: default json payload with application
+        val payloadKey = getRandomString()
+        val payloadTransfer = PayloadTransfer.getDefaultJsonPayloadTransfer().apply {
+            key = payloadKey
+            application = applicationName
+            ttlseconds = 0
+        }
+
+        // when: POST module-storage endpoint is called
+        val exception = shouldThrowExactly<ApiException> {
+            cacheApi.postStorageCache(payloadTransfer, apiKey) }
+
+        // then: Expire time exception is thrown
+        assertSoftly {
+            exception.statusCode shouldBe INTERNAL_SERVER_ERROR.value()
+            exception.responseBody shouldContain "\"path\":\"/storage\""
+            exception.responseBody shouldContain "\"message\":\"ERR invalid expire time in setex"
+        }
+    }
+
+    should("not throw an exception when ttlsecond is null and config ttlseconds are present") {
```

Comment:
What if we don't have ttlSeconds in the config, but it is included in payloadTransfer or not? Could you please add this test case?